### PR TITLE
[CP-1122] I can delete multiple threads by delete button in the selection manager

### PR DESCRIPTION
**Pull request by wercix**
Jira: [CP-1122]

**Description**
- Add selection manager on messages
- Removed checkbox tooltip for now, will be added in CP-1306
<details>
<summary><b>Screenshots</b></summary>

Screen Shot 2022-05-18 at 13 30 55

</details>

[CP-1122]: https://appnroll.atlassian.net/browse/CP-1122?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dkarski — COMMENTED** [review]
- bug with thread placeholderv (video attached)
- bug in reducer with messages, a while application throw blank screen
- no consequence in delete thread (production/ internal release) (screen attached)


https://user-images.githubusercontent.com/17147149/169032824-7210c40a-27e4-430a-ac67-7135ba1e7f6e.mov

Zrzut ekranu 2022-05-18 o 13 50 23
Zrzut ekranu 2022-05-18 o 13 53 08

**Comment by igorbezsmertnyi** [comment]
@wercix I have noticed a minor logical issue. After the contact is deleted the checkbox is automatically selected.

https://user-images.githubusercontent.com/21984930/169480649-306c63f6-b704-4d5b-830b-88d161d26b47.mov

d

**Comment by dkarski** [comment]
> @wercix I have noticed a minor logical issue. After the contact is deleted the checkbox is automatically selected.
> 
>  Screen.Recording.2022-05-20.at.9.50.24.AM.mov 
> d

This bug has reported as https://appnroll.atlassian.net/browse/CP-1129

File: packages/app/src/messages/components/messages/messages.component.tsx [hunk 1]
```diff
@@ -128,24 +127,15 @@ const Messages: FunctionComponent<Props> = ({
   deletingState,
   hideDeleteModal,
 }) => {
-  const [_, setThreadsPaginationOffset] = useState<
-    PaginationBody["offset"] | undefined
-  >(0)
-
   useEffect(() => {
+    console.log("messageLayoutNotifications:", messageLayoutNotifications)
```
Comment (dkarski) [inline packages/app/src/messages/components/messages/messages.component.tsx:131]: to delete 😉

Comment (wercix) [reply packages/app/src/messages/components/messages/messages.component.tsx]: Fixed in [c9ebc68](https://github.com/mudita/mudita-center/pull/722/commits/c9ebc68786cb7354dcfc129f147c984f311181c5)

**Review by dkarski — APPROVED** [review]
LGTM ✅

**Review by igorbezsmertnyi — APPROVED** [review]

**Pull request merged**